build/Dockerfile.debian.dev

```diff
@@ -0,0 +1,61 @@
+# Purpose of this container image is to ship pocket binary with additional
+# tools such as dlv, curl, etc.
+
+ARG TARGET_GOLANG_VERSION=1.18
+
+FROM golang:${TARGET_GOLANG_VERSION}-bullseye AS builder
+
+ENV PROTOC_VERSION 3.19.4
+ENV GOLANG_PROTOBUF_VERSION 1.28
+ENV MOCKGEN_VERSION 1.6.0
+ENV PATH $PATH:$GOPATH/bin
+
+### Install dependencies
+# Debian packages
+RUN apt-get update -qq && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential zip wget ca-certificates curl net-tools && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt
+
+# protoc
+RUN set -eux; \
+    arch="$(dpkg --print-architecture)"; arch="${arch##*-}"; \
+    url=; \
+    case "$arch" in \
+    'amd64') \
+    url="https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip"; \
+    ;; \
+    'arm64') \
+    url="https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-aarch_64.zip"; \
+    ;; \
+    *) echo >&2 "error: unsupported architecture '$arch' (likely packaging update needed)"; exit 1 ;; \
+    esac; \
+    env; \
+    echo "$url"; \
+    wget -O protoc.zip "$url" --progress=dot:giga; \
+    unzip -q protoc.zip -d /usr/local/; \
+    protoc --version
+
+# protoc-gen-go
+RUN go install "google.golang.org/protobuf/cmd/protoc-gen-go@v${GOLANG_PROTOBUF_VERSION}"; \
+    protoc-gen-go --version
```

Comment:
Is this common practice so the version is in the logs?

Reply:
I see no reason why not to, just to make sure the binary is executable and fail on build step if it is not.